src/console_reporter.cc

```diff
@@ -113,4 +121,16 @@ void ConsoleReporter::PrintRunData(const Run& result) {
               result.report_label.c_str());
 }
 
+TimeUnitMultiplier ConsoleReporter::getTimeUnitAndMultiplier(TimeUnit unit) {
```

Comment:
right, but they should now scale appropriately given the user has requested a particular time scale.

they should maybe have a field for the time unit added too...